zenhub/help.md

```diff
@@ -1,14 +1,12 @@
 # Description
 
-The [Zenhub](https://www.zenhub.com) plugin allows for agile project management in GitHub. This plugin will allow you to manage issues, epics, and pipelines in a ZenHub enable Git repository.
-
+The [Zenhub](https://www.zenhub.com) plugin allows for agile project management in GitHub. This plugin will allow you to manage issues, epics, and pipelines in a ZenHub enabled Git repository.  Kaban boards to plan projects, reports to track progress and give predicted release dates.
```

Comment:
the last sentence in this description doesn't really make sense - it's more of a sentence fragment